fda/datasets/_samples_generators.py

```diff
@@ -49,3 +49,57 @@ def make_gaussian_process(n_samples: int=100, n_features: int=100, *,
     y = random_state.multivariate_normal(mu, covariance, n_samples)
 
     return FDataGrid(sample_points=x, data_matrix=y)
+
+
+def make_sinusoidal_process(n_samples: int=15, n_features: int=100, *,
+                            start: float=0., stop: float=1., period: float=1.,
+                            phase_mean: float=0., phase_std: float=.6,
+                            amplitude_mean: float=1., amplitude_std: float=.05,
+                            error_std: float=.2, random_state=None):
+
+    """Generate sinusoidal proccess.
+
+    Each sample :math:`x_i(t)` is generated as:
+
+    .. math::
+
+        x_i(t) = \\alpha_i \sin(\omega t + \phi_i) + \epsilon_i(t)
+
+
+    where :math:`\omega=\\frac{2 \pi}{\\text{period}}`. Amplitudes
+    :math:`\\alpha_i` and phases :math:`\phi_i` are normally distributed.
+    :math:`\epsilon_i(t)` is a gaussian white noise process.
+
+    Args:
+        n_samples: Total number of samples.
+        n_features: Points per sample.
+        start: Starting point of the samples.
+        stop: Ending point of the samples.
+        period: Period of the sine function.
+        phase_mean: Mean of the phase.
+        phase_std: Standard deviation of the phase.
+        amplitude_mean: Mean of the amplitude.
+        amplitude_std: Standard deviation of the amplitude.
+        error_std: Standard deviation of the gaussian Noise.
+        random_state: Random state.
+
+    Returns:
+        :class:`FDataGrid` object comprising all the samples.
+
+    """
+
+    sklearn.utils.check_random_state(random_state)
```

Comment:
You should assign the result.

Reply:
I thought it works like seed, thanks for the correction. Fixed in e72bf90.